diff --git a/src/lib/uTop_main.ml b/src/lib/uTop_main.ml
index 1f873cd46..bb1403f18 100644
--- a/src/lib/uTop_main.ml
+++ b/src/lib/uTop_main.ml
@@ -639,7 +639,7 @@ let rewrite_str_item pstr_item tstr_item =
 
 let type_structure env pstr =
 #if OCAML_VERSION >= (4, 14, 0)
-  let tstr, _, _, _, _ = Typemod.type_structure env pstr in
+  let tstr, _, _, _ = Typemod.type_structure env pstr in
 #elif OCAML_VERSION >= (4, 12, 0)
   let tstr, _, _, _ = Typemod.type_structure env pstr in
 #elif OCAML_VERSION >= (4, 08, 0)
